### ISSUE TITLE
FormLayout.Group children aren't vertical aligned correctly with TextField's error

### ISSUE THREAD

**Issue by Tiendq** [Shopify/polaris#230]
# Issue summary

```
<FormLayout>
  <FormLayout.Group>
    <TextField
      type="number"
      label="Minimum order"
      error="Test"
    />
    <TextField
      type="number"
      label="Maximum order"
    />
  </FormLayout.Group>
</FormLayout>
```

## Expected behavior

`TextField`s must be still vertical aligned correctly, e.g. top

## Actual behavior

Center aligned.

## Steps to reproduce the problem

1. Go to https://polaris.shopify.com/components/forms/form-layout#navigation
2. Add `error="Test"` to a TextField

## Specifications

- Polaris version: current
<img width="908" alt="screen shot 2017-11-23 at 12 05 48 pm" src="https://user-images.githubusercontent.com/815910/33159639-4f256c5a-d047-11e7-8a1b-9f72a034a64f.png">

**Comment by ry5n** [comment]
Yup, I can reproduce this. I’m not sure why we have the outer container (flexbox) vertically centering the items. Looking into it.

**Comment by sp4cecat** [comment]
I think that I solved this by overriding the CSS to align them to flex-start.

**Comment by Tiendq** [comment]
@sp4cecat it is what I did too, it's a trivial bug, but it still need to be fixed to not make others surprise :)

**Comment by dfmcphee** [comment]
This was fixed as of 1.11.0: https://github.com/Shopify/polaris/blob/master/CHANGELOG.md#1110---2018-02-13
